Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -0,0 +1,287 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingIndirect
+  "Indirect cooling energy transfer station for district energy systems"
+  extends Buildings.Fluid.Interfaces.PartialFourPort(
+    redeclare package Medium1 = Medium,
+    redeclare package Medium2 = Medium);
```

Comment:
It can be finalized. 
```
redeclare final package Medium1 = Medium,
redeclare final package Medium2 = Medium);
```

Reply:
Done. I have also finalized other parameters in `CoolingIndirect.mo` that have been propagated.